ppr-ui/src/components/tables/common/TableRow.vue

```diff
@@ -166,7 +166,7 @@
             <span>Edit</span>
           </v-btn>
           <v-btn
-            v-else-if="isRepairersLien(item)"
+            v-else-if="isRepairersLien(item) && !isDischarged(item)"
```

Comment:
I think this should be `isRepairersLien(item) && isActive(item)` because that way it covers if it is expired as well ?

Reply:
updated to isActive